pax-jdbc-itest/src/test/java/org/ops4j/pax/jdbc/test/pool/PoolNarayanaTest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.ops4j.pax.jdbc.test.pool;
+
+import javax.inject.Inject;
+
+import org.junit.Test;
+import org.ops4j.pax.exam.Configuration;
+import org.ops4j.pax.exam.Option;
+import org.ops4j.pax.jdbc.pool.common.PooledDataSourceFactory;
+import org.ops4j.pax.jdbc.test.AbstractJdbcTest;
+
+public class PoolNarayanaTest extends AbstractJdbcTest {
+
+    @Inject @org.ops4j.pax.exam.util.Filter("(pool=narayana)(xa=true)")
+    PooledDataSourceFactory xaPool;
+    
+    @Inject @org.ops4j.pax.exam.util.Filter("(pool=narayana)(xa=false)")
+    PooledDataSourceFactory pool;
+    
+    @Configuration
+    public Option[] config() {
+        return new Option[] {
+            regressionDefaults(), //
+            poolDefaults(), //
+            mvnBundle("com.h2database", "h2"), //
+            mvnBundle("org.jboss", "jboss-transaction-spi"), //
```

Comment:
@grgrzybek I add this itest for the pax-jdbc-pool-narayana but it fails with
```
00:16:15.406 ERROR {Framework Event Dispatcher: org.eclipse.osgi.internal.framework.EquinoxEventPublisher@20140db9} [org.ops4j.pax.exam.nat.internal.NativeTestContainer] : Framework ERROR event org.osgi.framework.FrameworkEvent[source=org.ops4j.pax.jdbc.pool.narayana_1.4.4.SNAPSHOT [27]]
org.osgi.framework.BundleException: Could not resolve module: org.ops4j.pax.jdbc.pool.narayana [27]
  Unresolved requirement: Import-Package: org.jboss.tm

        at org.eclipse.osgi.container.Module.start(Module.java:444) ~[org.eclipse.osgi-3.11.2.v20161107-1947.jar:?]
        at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1620) ~[org.eclipse.osgi-3.11.2.v20161107-1947.jar:?]
        at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1600) ~[org.eclipse.osgi-3.11.2.v20161107-1947.jar:?]
        at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1571) ~[org.eclipse.osgi-3.11.2.v20161107-1947.jar:?]
        at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1514) ~[org.eclipse.osgi-3.11.2.v20161107-1947.jar:?]
        at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1) ~[org.eclipse.osgi-3.11.2.v20161107-1947.jar:?]
        at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:230) [org.eclipse.osgi-3.11.2.v20161107-1947.jar:?]
        at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:340) [org.eclipse.osgi-3.11.2.v20161107-1947.jar:?]
```
I added
```
mvnBundle("org.jboss", "jboss-transaction-spi")
```
here and it looks like the bundle has not been scanning.
Can you take a look and give some hints please ?

Reply:
OK, it looks like we need to use the org.jboss.narayana:narayana-osgi-jta here.